db/migrate/20170325135953_remove_unused_indexes_sgl2017.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveUnusedIndexesSgl2017 < ActiveRecord::Migration
+  def change
+    # PRE SGL 2017 index cleanup
+    remove_index :recents, :classification_id
+    remove_index :set_member_subjects, :priority
+    remove_index :oauth_applications, [:owner_id, :owner_type]
+    remove_index :subject_queues, [:workflow_id, :user_id]
+    remove_index :subjects, :activated_state
```

Comment:
This one seems to not being hit in the selector query [here](https://github.com/zooniverse/Panoptes/blob/91ec95d77b682dd4e0715d9bdff64b225e4fb949/lib/subjects/selector.rb#L79) as the db is most likely running a filter over the small results instead. 

Also this could be actively used if we start looking at rake task to clean up orphaned subjects. It's currently 196MB. I'm keen to remove it and add it back in if we start cleaning up orphans on a schedule.